### PR TITLE
Support to Check the HTTP Response Text

### PR DESCRIPTION
**Pull request by haoel**
This PR address the issue  #76  

simply copy the same feature from `shell` and `ssh` probe.

### CODE REVIEW

**Review by nullsimon — COMMENTED** [review]
can we remove this blank line

Screen Shot 2022-05-13 at 20 32 06

File: README.md [hunk 1]
```diff
@@ -387,6 +387,9 @@ http:
     success_code:
       - [200,206] # the code >=200 and <= 206
       - [300,308] # the code >=300 and <= 308
+    # Response Checking
+    contain: "success" # if not contain this string, the server is failed.
```
Comment (proditis) [inline README.md:391]:
```suggestion
    contain: "success" # response body must contain this string, if not the probe is considered failed.
```

File: README.md [hunk 2]
```diff
@@ -387,6 +387,9 @@ http:
...
+    not_contain: "failure" # if contain this string, the server is failed.
```
Comment (proditis) [inline README.md:392]:
```suggestion
    not_contain: "failure" # response body must NOT contain this string, if it does the probe is considered failed.
```

**Review by proditis — COMMENTED** [review]
looks good to me, these two minor suggestions can be ignored if you like (i'll address them at a later PR along with other documentation fixes)

**Review by nullsimon — APPROVED** [review]

**Review by zhao-kun — APPROVED** [review]

**Pull request merged**